hack/demo.sh

```diff
@@ -48,16 +48,21 @@ spec:
   configuration:
     exclude:
     - not_useful
-    - test:conftest-clair
+    include:
+    - slsa_provenance_available
```

Comment:
Specifying one include like this means use *only* that item. Is that what you're intending here?

Reply:
I miss understand the include here, so deleting it now.